elide-contrib/elide-dynamic-config-helpers/src/main/java/com/yahoo/elide/contrib/dynamicconfighelpers/ElideHjsonUtil.java

```diff
@@ -0,0 +1,177 @@
+import org.everit.json.schema.loader.SchemaLoader;
+import org.hjson.JsonValue;
+import org.json.JSONObject;
+import org.json.JSONTokener;
+
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.URL;
+import java.nio.charset.Charset;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+/**
+ * Util class for HJSON parsing
+ */
+@Slf4j
+public class ElideHjsonUtil {
+
+    public static final String SCHEMA_TYPE_TABLE = "table";
+    public static final String SCHEMA_TYPE_SECURITY = "security";
+    public static final String SCHEMA_TYPE_VARIABLE = "variable";
+
+    public static final String ELIDE_TABLE_VALIDATION_SCHEMA = "elideTableSchema.json";
+    public static final String ELIDE_SECURITY_SCHEMA = "elideSecuritySchema.json";
+    public static final String ELIDE_VARIABLE_SCHEMA = "elideVariableSchema.json";
+
+    public static final String INVALID_ERROR_MSG = "Incompatible or invalid config";
+    public static final String HTTP_PREFIX = "http";
+    public static final String CHAR_SET = "UTF-8";
+    public static final String NEW_LINE = "\n";
+
+   /**
+    * HJSON to JSON string conversion.
+    * @param hjson hjson string
+    * @return json string
+    */
+    public static String hjsonToJson(String hjson) {
+        return JsonValue.readHjson(hjson).toString();
+    }
+
+    /**
+     * Config file reader. Supports HTTP or Local Filesystem. 
+     * @param filePath Path to File
+     * @return contents of the file
+     * @throws Exception
+     */
+    public static String readConfigFile(String filePath) throws Exception {
+
+        BufferedReader reader = null;
+        try {
+            reader = (filePath.startsWith(HTTP_PREFIX)
+                            ? getHttpFileReader(filePath) : getLocalFileReader(filePath));
+            return readFileContent(reader);
+        } catch (Exception e) {
+            throw e;
+        } finally {
+            reader.close();
+        }
+    }
+
+    /**
+     * Validates JSON to Schema.
+     * @param schemaType type of schema
+     * @param jsonConfig json string
+     * @return whether json validation passed
+     */
+    public static boolean validateDataWithSchema(String schemaType, String jsonConfig) {
+
+        try {
+            JSONObject schemaObj = schemaToJsonObject(schemaType);
+            JSONObject data = new JSONObject(new JSONTokener(jsonConfig));
+
+            Schema schema = SchemaLoader.load(schemaObj);
+            schema.validate(data);
+            return true;
+        } catch (Exception e) {
+            e.printStackTrace();
+            log.error(e.getMessage());
+            return false;
+        }
+    }
+
+    /**
+     * Null check for the string.
+     * @param input string to validate
+     * @return whether string is null or not
+     */
+    public static boolean isNull(String input) {
```

Comment:
Lets rename this to isNullOrEmpty. Also we could just do 
`return (input == null || input.trim().length() == 0)` instead `return (input == null || input.trim().length() == 0) ? true : false`